.github/CONTRIBUTING.md

```diff
@@ -1,34 +1,90 @@
 # Contributing to Kitura
 
-We welcome contributions, and request you follow these guidelines.
+This document contains information and guidelines about making contributions. For help, contact us on [Slack](http://swift-at-ibm-slack.mybluemix.net/).
 
- - [Raising issues](#raising-issues)
+We love contributions to the Kitura project, but request you follow the guidelines below.
+
+ - [Getting Started](#getting-started)
+ - [Pull Requests](#pull-requests)
+ - [Coding Style](#coding-style)
  - [Contributor License Agreement](#contributor-license-agreement)
- - [Coding Standards](#coding-standards)
+ - [Asking Questions](#asking-questions)
+ - [Reporting issues](#reporting-issues)
+ - [Additional Resources](#additional-resources)
+
+
+## Getting Started
+
+Please ensure you follow [the Kitura coding standards](https://github.com/IBM-Swift/Kitura/blob/master/Documentation/CodeConventions.md). This keeps the source code tidy, and allows [Swift Lint]((https://github.com/realm/SwiftLint)) to work with the project.
+
+**If you are new:** We have created the label ["good starting issue"](https://github.com/IBM-Swift/Kitura/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to help new members of the community get involved with Kitura quickly and effectively. These issues include documentation changes, test cases, minor bug fixes, logging, renaming and refactoring.
```

Comment:
The label is `good first issue` isn't it, not `good starting issue`?

Reply:
I don't like the title `If you are new` how about `New to the community?` then you could have it as a minor title rather than embedded bold text...?

Reply:
✅